src/docs/sdk/performance/span-data-conventions.mdx

```diff
@@ -43,6 +43,8 @@ Below describes the conventions for the Span interface for the `data` field on t
 | `frames.slow`     | int | The number of slow frames rendered during the lifetime of the span.     |   `2`   |
 | `frames.frozen`   | int | The number of frozen frames rendered during the lifetime of the span.   |   `1`   |
 | `frames.delay`    | number | The sum of all delayed frame durations in seconds during the lifetime of the span. For more information see <Link to="/sdk/performance/frames-delay/">frames delay</Link>.   |   `1.3246`   |
+| `contributes_ttid`    | boolean |  Whether the span execution contributed to the TTID (time to initial display) metric.  |   `true`   |
```

Comment:
to preserve data, SDKs should probably only set the flag if it's value is `true`.

Reply:
could we add a namespace for this? like `mobile.contributes_ttid`?

Reply:
Actually I guess `ttid` could apply for gaming/browser in the future, could we make it `ui.contributes_ttid` instead?

Reply:
I agree with `ui.contributes_ttid`. I think we could start a new category as mobile for this.